developers/ide/eclipse.md

```diff
@@ -14,38 +14,53 @@ For now follow these steps:
 
 ### Eclipse IDE Setup
 
-1. Install the latest "Eclipse IDE for Java Developers", the installer [can be downloaded from the Eclipse web site](https://wiki.eclipse.org/Eclipse_Installer)
-1. Launch Eclipse and install Bndtools from the Eclipse Marketplace: Help -> Eclipse Marketplace -> Find "bndtools" -> Install:
+This guide focuses on Add-on (binding) development.
 
-   ![bndtools from marketplace](images/ide_setup_eclipse_market.png)
-   (Accept License and the Security warning about the software is not signed)
+1. Download the "Eclipse Installer": [can be downloaded from the Eclipse web site](https://wiki.eclipse.org/Eclipse_Installer)
+1. Launch the Installer and select `Advanced Mode...` on the menu (3 bars) on the right, then select `Eclipse IDE for Java Developers` then `Next>`
+1. Select `Github Projects > openHAB > openHAB Development`. 
+    
+    Do *NOT* select other projects for now.
 
-   Restart Eclipse to activate Bndtools.
-1. Using `git` from the command line, checkout the openHAB demo application: `git clone --depth=1 https://github.com/maggu2810/openhab-demo`
-1. Import the demo application from Eclipse: File -> Import -> Maven -> Existing Maven Projects -> select the root directory of the demo application project -> Finish.
+   ![select projects](images/ide_setup_eclipse_projects.png)
```

Comment:
This image is missing :)
(hence the build failure!)
Please also use make relative URLs like this: `./images/ide_setup_eclipse_projects.png`